### PR TITLE
Add testnet2 for StarkNet and minor issues

### PR DESCRIPTION
**Pull request by ginstagram**
## Description
Add testnet2 for StarkNet
Add rel notes
Fix contact form link
Fix Ethereum API docs link

## Related Issue
https://github.com/chainstack/docs/issues/566

### CODE REVIEW

File: docs/api/ethereum/eth_getlogs.md [hunk 1]
```diff
@@ -26,10 +26,10 @@ Ethereum API method that returns an array of all logs matching a given filter ob
     * `earliest` — the genesis block.
     * `pending` — the pending state and transactions block.
   * `address` - (optional) Contract address, or a list of addresses from which logs should originate.
-  * `topics` - (optional) Array of `DATA` topics. Topics are order-dependent. Go here to learn more about topics.
+  * `topics` - (optional) Array of `DATA` topics. Topics are order-dependent. [See Ethereum official documentation](https://ethereum.org/en/developers/docs/apis/json-rpc/#ops-deployments-custom) to learn more about topics.
```
Comment (akegaviar) [inline docs/api/ethereum/eth_getlogs.md:29]: Let's keep "See" and the linked part separate as we currently do.

eg `See [Ethereum official documentation]`.

File: docs/operations/starknet/networks.md [hunk 2]
```diff
@@ -12,8 +12,9 @@ Chainstack supports joining the following StarkNet networks:
 
 * Mainnet — public StarkNet mainnet. L2 to the Ethereum mainnet.
 * Testnet — public StarkNet testnet. L2 to the Ethereum Goerli testnet.
+* Testnet2 — alternative public StarkNet testnet. L2 to the Ethereum Goerli testnet.
 
-For development purposes on the testnet, fund your accounts with [L2 Goerli ETH](https://faucet.goerli.starknet.io/).
+For development purposes on the testnet and testnet2, fund your accounts with [L2 Goerli ETH](https://faucet.goerli.starknet.io/).
```
Comment (akegaviar) [inline docs/operations/starknet/networks.md:17]: @ginstagram @soos3d Will this fund the account on testnet2?

Also, the faucet doesn't seem to be working.

Check what the deal with the testnet1 and tesnet2 faucets is and then either remove the mention here or add information that works for either/both testnets.

We need high precision working instructions.

File: docs/operations/starknet/networks.md [hunk 3]
```diff
@@ -12,8 +12,9 @@ Chainstack supports joining the following StarkNet networks:
 
 * Mainnet — public StarkNet mainnet. L2 to the Ethereum mainnet.
 * Testnet — public StarkNet testnet. L2 to the Ethereum Goerli testnet.
+* Testnet2 — alternative public StarkNet testnet. L2 to the Ethereum Goerli testnet.
```
Comment (akegaviar) [inline docs/operations/starknet/networks.md:15]: Needs better description.

It's not clear why it's an alternative. https://community.starknet.io/t/calling-for-proposals-for-naming-starknets-testnets/2287

It's a less congested test network for dapp developers. something like that.

**Review by akegaviar — CHANGES_REQUESTED** [review]
Changes please. Needs some research.

File: docs/platform/supported-cloud-hosting-providers.md [hunk 4]
```diff
@@ -21,5 +21,5 @@ Chainstack also supports private hosting options. See [Supported hosting options
 ::: tip
 Chainstack is always working on introducing support for other regions for cloud deployment.
 
-Want to request a cloud provider or region support? <a href="https://chainstack.com/contact/" target="_blank">Contact us</a>.
+Want to request a cloud provider or region support? <a href="https://ideas.chainstack.com/feature-requests" target="_blank">Contact us</a>.
```
Comment (akegaviar) [inline docs/platform/supported-cloud-hosting-providers.md:24]: This and other case like this—it's no longer "Contact us" though. It's more of a check and vote on the existing feature requests or submit a new one. < Something like that.

File: docs/release-notes.md [hunk 5]
```diff
@@ -8,6 +8,14 @@ meta:
 
 # Release notes
 
+## Chainstack 3.8.1
+
+*November 14*
+
+### What's new
+
+* **Protocols**. [StarkNet](/operations/starknet/networks) testnet2 support for elastic and dedicated nodes.
```
Comment (akegaviar) [inline docs/release-notes.md:17]: change to `testnet2 (Goerli2)`.

**Review by akegaviar — CHANGES_REQUESTED** [review]
Changes please

**Review by akegaviar — APPROVED** [review]

**Comment by conor2e** [comment]
@akegaviar @ginstagram I'm approving this PR but going forward, we should clarify with the support team which links we should be promoting to users and consistently clarify our instructions:

- They can view a list of existing feature requests on the ideas website and can "create a post": https://ideas.chainstack.com/feature-requests. 
- They can "submit a request" on the support website here: https://support.chainstack.com/hc/en-us/requests/new
- They can also contact support on the main website, or "_Reach out to us for custom solutions, feature requests, feedback, and any questions you may have regarding Chainstack’s pricing and infrastructure._" here: https://chainstack.com/contact/
- We also ask them to contact us and send them to the support website home page, where it's not really clear from there how to contact us: https://support.chainstack.com/hc/en-us

We should also clarfiy:

- Do they need to create a support account to be able to submit a request on the support website?
- Do they need to create an "ideas.chainstack.com" account to create a post (are these separate accounts)?

**Review by conor2e — APPROVED** [review]

File: docs/release-notes.md [hunk 6]
```diff
@@ -8,6 +8,14 @@ meta:
 
 # Release notes
 
+## Chainstack 3.8.1
+
+*November 14*
```
Comment (conor2e) [inline docs/release-notes.md]: @ginstagram You left the year out of the date here. Please fix in the next release.

**Pull request merged**